src/Components/CategoryLayout.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import Bell from "../assets/icons/Notification.svg"
+import Category from "../assets/icons/Category.svg"
+import DefaultPic from "../assets/images/profile_pic.png"
+import Info from "../assets/icons/Info.svg"
+
+import { IoLogOutOutline } from "react-icons/io5";
+import { RxCross2 } from "react-icons/rx";
+
+interface CategoryLayoutProps {
+    children: React.ReactNode;
+    organization: string;
+    category: string;
+}
+
+export default function CategoryLayout({ children, organization, category }: CategoryLayoutProps) {
+    return (
+        <div>
+            <header className="flex justify-between items pl-[82px] pr-[42px] pt-[17px] pb-[27px]">
+                <div className='flex flex-col justify-center'>
+                    <p className='text-[13px]'>Good morning 👋</p>
+                    <h3 className='text-[16px] font-bold'>{organization}</h3>
+                </div>
+                <div className='flex items-center gap-10'>
+                    <img
+                        src={Bell}
+                        className='w-6 h-6'
+                    />
+                    <div className='flex items-center gap-2'>
+                        <div className='p-3 bg-eclipse rounded-full'>
+                            <img src={DefaultPic} className="w-8 h-8" />
+                        </div>
+                        <h1 className='font-bold'>Moni Roy</h1>
```

Comment:
I will recommend this to be wrapped in a button or link too. CLicking on this should redirect user to Profile page